src/Covenant/JSON.hs

```diff
@@ -0,0 +1,517 @@
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
+
+module Covenant.JSON where
```

Comment:
Since (most of) the identifiers in this module will be publically-visible, they should at least have `@since`s for now. We can do a proper export list later.